src/main/webapp/app/controllers/settings/emailTemplateRepoController.js

```diff
@@ -120,14 +120,13 @@ vireo.controller("EmailTemplateRepoController", function ($controller, $scope, $
             container: '#email-templates'
         });
 
-        EmailTemplateRepo.listen(function (data) {
-            $scope.resetEmailTemplates();
+        EmailTemplateRepo.listen(function (/* data */) {
```

Comment:
I think that it would be ideal to remove the `/* data */` rather than leave unused commented out code.